radio/src/opentx.cpp

```diff
@@ -1064,9 +1068,8 @@ void getADC()
   DEBUG_TIMER_STOP(debugTimerAdcRead);
 
   for (uint8_t x=0; x<NUM_ANALOGS; x++) {
-    uint16_t v;
```

Comment:
If you really need a different type (see previous comment), just use `auto` for the type instead of doubling the code.

Reply:
Thx. Learned again something. But as I changed now to 32-bit always, it can stay as uint32_t.